.github/workflows/build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         id: generate-ldflags
         run: |
           project="$(go list -m)"
-          echo "::set-output name=ldflags::"-X \'$project/version.GitDescribe=v${{ needs.get-product-version.outputs.product-version }}\'""
+          echo "ldflags="-X \'$project/version.GitDescribe=v${{ needs.get-product-version.outputs.product-version }}\'"" >> $GITHUB_OUTPUT
```

Comment:
I have no idea if this will be broken with the extra `=` in there and have no way of testing it, so... :man_shrugging: I guess we'll find out whether this works the next time we do a release.